components/Player/index.vue

```diff
@@ -13,65 +13,65 @@
+            <!-- The portal is either being created or starting -->
+            <div v-if="portalStatus === 'creating' || portalStatus === 'starting'" class="player-message">
                 <h1 class="title">
-                    And so it begins...
+                    We're {{ portalStatus }} your browser now
                 </h1>
                 <p class="body">
-                    Add a friend or two to get the party started! Once someone else joins this room, we'll queue up your room for a virtual browser
+                    Normally this takes a couple seconds, hang tight!
+                    <br>
+                    If you have any issues either refresh the page or ask the room owner to restart the browser
                 </p>
             </div>
-            <!-- Portal is not open but there is more than one member in the room -->
-            <div v-else-if="(portalStatus === 'waiting' || portalStatus === 'closed') && room.members.length > 1" class="player-message">
+            <!-- The portal has been created and the stream between the client and the aperture is being established -->
+            <div v-else-if="portalStatus === 'open'" class="player-message">
                 <h1 class="title">
-                    Hold your rabbits...
+                    Strap in...
                 </h1>
                 <p class="body">
-                    We're waiting for the right gears to align, so we can get your browser ready - hold tight!
-                    <br>
-                    If this is taking a little long, restart the room
+                    Everything is ready - we're just getting the stream between the browser and you started
                 </p>
             </div>
-            <!-- A portal has been placed into the queue -->
-            <div v-else-if="portalStatus === 'in-queue'" class="player-message">
+            <!-- Something wrong happened while starting the portal -->
+            <div v-else-if="portalStatus === 'error'" class="player-message">
                 <h1 class="title">
-                    You're in the queue!
+                    Oh, no!
                 </h1>
                 <p class="body">
-                    There are a couple rooms in front of you waiting for a virtual browser, please grab your popcorn!
+                    Something bad happened that isn't letting this instance from starting a virtual browser.
+                    <br>
+                    We'll try to recover, but if this is stuck for too long, restart the room
                 </p>
             </div>
-            <!-- The portal is either being created or starting -->
-            <div v-else-if="portalStatus === 'creating' || portalStatus === 'starting'" class="player-message">
+            <!-- A portal has been placed into the queue -->
+            <div v-else-if="queueStatus !== null" class="player-message">
```

Comment:
I'm a bit unsure about this one. Don't you think it would be better to use `portalStatus === 'in-queue'` and in body use a `v-if="queueStatus !== null` and `v-else`?

Reply:
That actually was what I did originally, however an updatePortal is not actually sent for in queue. 

When you get the queueStatus from api, the portal doesn't actually exist yet so there's nothing to update the status of, thus we never get an updatePortal command for it.